quotest/quotest.cpp

```diff
@@ -911,7 +911,7 @@ void TestManager::conclude()
     // Clean up the room (best effort)
     auto* room = testSuite->room();
     room->setTopic({});
-    room->localUser()->rename({});
+    room->connection()->user()->rename({});
```

Comment:
No idea why the original code went through `room`, it wasn't needed.
```suggestion
    c->user()->rename({});
```